tests/example_app/example/about/our_team/bp_team.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+
+bp_aboutTeam = Blueprint("our_team", __name__,
```

Comment:
Can you make similar changes to the previous comment except using the word `team` instead of `story`?